Readme.md

```diff
@@ -33,6 +33,11 @@ $ ember install ember-outside-click
 
 [See the demo code][3].
 
+#### Excluded classes
+
+Add the `excludedClasses` array property to either the block form or subclass to exclude classes
+from the outside click behavior.
+
```

Comment:
Awesome, please link to the subclass example, and add an example below this line for the block form. Like:

```hbs
{{#outside-click excludedClasses=(array 'excluded-class' 'special-toolbar')}}{{/outside-click}}
```